### PR TITLE
reorganise solid with sub-folders

### PR DESCRIPTION
**Pull request by bourgeoa**
- created `solid-rest-api` folder and moved the fetch/N3 solid example to that branch
- updated index.html's when needed
- updated symbolic links
- renamed solid/all.html to solid/index.html
- solid-file-client updated to most of @NoelDeMartin comments.
  closed agreed comments and leave the other ones open for discussion (related to pim:storage)
- did not change the solid/README.md

### CODE REVIEW

File: solid/solid.js [hunk 1]
```diff
@@ -1,275 +0,0 @@
-// You can find the basic Solid concepts explained in the Glossary.md file, inline comments talk about
```
Comment (NoelDeMartin) [inline solid/solid.js]: It seems like you have removed this file entirely, instead of moving it to `solid/solid-rest-api`. I guess that was a mistake?

File: solid/solid-file-client/index.html [hunk 2]
```diff
@@ -37,8 +37,8 @@ <h2>Your tasks</h2>
 
         <script src="https://cdn.jsdelivr.net/npm/@inrupt/solid-client-authn-browser@1.11.2/dist/solid-client-authn.bundle.js"></script>
         <script src="https://cdn.jsdelivr.net/npm/solid-file-client@1.2.5/dist/window/solid-file-client.bundle.js"></script>
-        <script src="solid.js"></script>
-        <script src="main.js"></script>
+        <script src="./solid.js"></script>
```
Comment (NoelDeMartin) [inline solid/solid-file-client/index.html:40]: Was this change necessary? I'd rather use the same style in all examples.

**Review by NoelDeMartin — COMMENTED** [review]
Does this PR superseed #14 then? re-structuring the folders and updating the `solid-file-client` example are different things, so I'm fine doing them in separate PRs and merge each one whenever they are ready.

File: solid/solid-file-client/index.html (hunk 2 above)
Comment (bourgeoa) [reply solid/solid-file-client/index.html:40]: removed the change

**Comment by bourgeoa** [comment]
I propose to close PR #14  and only keep #16

- @NoelDeMartin I think I made all your changes.
- Remain to do : the documentation organisation/update
  - I left README.md and Glossary.md in hello/solid/ as they  relate mostly to SOLID. A very small part concerns fetch() or solidFileClient()
  - what do you think of  ? 
    - keeping the general SOLID documentation in the solid folder (the solid/index.html will link to the solid/ folder
    - the specific implementation one in solid-rest-api/ and solid-file-client/. The index.html will link to there respective repository containing a small README.me with a link to --> solid/README.md --> Glossary.md

**Comment by NoelDeMartin** [comment]
Ok, I'm merging this as is and I'll open a new PR with a couple of tweaks.
